.github/workflows/examples.yml

```diff
@@ -119,20 +121,12 @@ jobs:
             NB_ARGS+=" --ignore=examples/check_dataset_parameter_coverage"
             NB_ARGS+=" --ignore=examples/conformer_energies"
             NB_ARGS+=" --ignore=examples/using_smirnoff_in_amber_or_gromacs"
-            # Remove when #1322 is merged
-            NB_ARGS+=" --ignore=examples/using_smirnoff_with_amber_protein_forcefield"
           fi
 
-          # Working on in #1325
-          NB_ARGS+=" --ignore=examples/virtual_sites/vsite_showcase.ipynb"
-
           # openmmforcefields incompatible with new toolkit
           NB_ARGS+=" --ignore=examples/external/swap_amber_parameters/swap_existing_ligand_parameters_with_openmmforcefields.ipynb"
 
-          # JM will do some smithing on this before 0.11.0 release
-          NB_ARGS+=" --ignore=examples/toolkit_showcase/toolkit_showcase.ipynb"
-
-          pytest $NB_ARGS examples
+          python -m pytest $NB_ARGS examples
```

Comment:
I can be talked out of this change; it's not needed here but I have a preference for running pytest and other tools as modules where possible